doc/ug/getting_started_verilator.md

```diff
@@ -40,8 +41,8 @@ The programs listed after `--meminit` are loaded into the system's specified mem
 ```console
 $ cd $REPO_TOP
 $ build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator \
-  --meminit=rom,build-bin/sw/device/boot_rom/boot_rom_sim_verilator.elf \
-  --meminit=flash,build-bin/sw/device/examples/hello_world/hello_world_sim_verilator.elf
+  --meminit=rom,build-out/sw/device/boot_rom/boot_rom_sim_verilator.elf \
+  --meminit=flash,build-out/sw/device/examples/hello_world/hello_world_sim_verilator.elf
```

Comment:
Why that change? `build-bin` is the output directory, `build-out` is the temporary build directory. Nothing should rely on a path in `build-bin`.